src/wkwebview/mod.rs

```diff
@@ -556,426 +318,143 @@ impl InnerWebView {
         });
 
         let frame = CGRect {
-          origin: window_position(if is_child { ns_view } else { webview }, x, y, h as f64),
+          origin: if is_child {
+            window_position(ns_view, x, y, h as f64)
+          } else {
+            CGPoint::new(x as f64, (0 - y - h as i32) as f64)
+          },
           size: CGSize::new(w as f64, h as f64),
         };
+        let webview: Retained<WryWebView> =
+          objc2::msg_send_id![super(webview), initWithFrame:frame configuration:&**config];
+        webview
+      };
+      #[cfg(target_os = "ios")]
+      let webview = {
+        let frame = ns_view.frame();
+        let webview = WKWebView::initWithFrame_configuration(webview, frame, &config);
+        webview
+      };
 
-        let _: () = msg_send![webview, initWithFrame:frame configuration:config];
+      #[cfg(target_os = "macos")]
+      {
         if is_child {
           // fixed element
-          webview.setAutoresizingMask_(NSViewMinYMargin);
+          webview.setAutoresizingMask(NSAutoresizingMaskOptions::NSViewMinYMargin);
         } else {
           // Auto-resize
-          webview.setAutoresizingMask_(NSViewHeightSizable | NSViewWidthSizable);
+          let options = NSAutoresizingMaskOptions(
+            NSAutoresizingMaskOptions::NSViewHeightSizable.0
+              | NSAutoresizingMaskOptions::NSViewWidthSizable.0,
+          );
```

Comment:
Nit: As I think you know, recent versions use `bitflags`, so this kind of thing shouldn't be necessary any more
```suggestion
          let options = NSAutoresizingMaskOptions::NSViewHeightSizable
              | NSAutoresizingMaskOptions::NSViewWidthSizable;
```